### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by jashgopani** [jashgopani/application-tracking-system#3] [Show and tell]
Please use this channel to share updates on a regular basis

**Comment by rohanajm7** [comment]
While using the project, there seem to be a few bugs. I will be raising an issues accordingly. Please do not forget to check them out!

**Comment by jashgopani** [comment]
I have been working on the [Issue #4](https://github.com/jashgopani/application-tracking-system/issues/4) and currently I have fixed the issue with some new code. Will now progress with the testing of the same.

**Comment by neerua08** [comment]
I am working on [Issue #13](https://github.com/jashgopani/application-tracking-system/issues/13). I have currently set up a Dockerfile. I am trying to set up MongoDB with docker.

**Reply by neerua08** [reply]
Docker seems to be mostly working.

**Comment by hemil26** [comment]
I am working on the Pylint issue, since our build is constantly failing in our recent pull request. I think the pylint workflow is incorrectly configured, I will be looking into it.

**Comment by neerua08** [comment]
Currently working on documentation of what features we added in phase 3.
